components.md

```diff
@@ -166,3 +166,29 @@ See `/transformers/optimus-prime.ts` to see how it is done.
 > For Content Transformers to work, we need to add them as modules to Nuxt.
 > Nuxt does this automatically if the nuxt modules are under `/modules`. You
 > can see how to add them as modules in `/modules/transform-module.ts`.
+
+## Data Fetching
+
+In our data fetching processes, we utilize Nuxt's operations such as `$fetch`,
+`useFetch`, and `useAsyncData`. To streamline the management and readability of
+tasks like specifying URLs, setting headers, and defining options in such
+operations, we prefer keeping them in a separate area. In Nuxt examples and as
```

Comment:
Didn't get this part, maybe "As it is automatically imported in Nuxt examples, we place them under the composables folder." or "We place them under the composables folder, as it is automatically imported in Nuxt examples." is better